nodes.py

```diff
@@ -879,13 +879,16 @@ class CLIPVisionLoader:
     @classmethod
     def INPUT_TYPES(s):
         return {"required": { "clip_name": (folder_paths.get_filename_list("clip_vision"), ),
+                             "device": (["GPU", "CPU"], ),
                              }}
     RETURN_TYPES = ("CLIP_VISION",)
     FUNCTION = "load_clip"
 
     CATEGORY = "loaders"
 
-    def load_clip(self, clip_name):
+    def load_clip(self, clip_name, device = "GPU"):
+        if device == "CPU":
+           comfy.clip_vision.ClipVisionModel.load_device = torch.device("cpu")
```

Comment:
this sets to CPU if input is CPU, but never sets to GPU for GPU setting, and never undoes this change

Reply:
Also, I don't think this code would even work?